agent-nodejs-8/test/agent_nodejs_test.go

```diff
@@ -113,4 +113,30 @@ var _ = Describe("NodeJS agent testing", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(code).To(Equal(0))
 	})
+
+	It("should contain a runnable yarn", func() {
+		// the default entrypoint for the image assumes if you supply more than
+		// one arg, you are trying to invoke the slave logic, so we have to
+		// override the entrypoint to run complicated commands for testing.
+		if strings.Contains(imageName, "rhel") {
```

Comment:
you need to add an import for the "strings" package.

Reply:
thanks